backend/src/index.ts

```diff
@@ -7,63 +9,67 @@ import dotenv from 'dotenv';
 
 dotenv.config();
 import * as Sentry from '@sentry/node';
-import { PORT, SENTRY_DSN, NODE_ENV, MONGO_URL, SITE_URL, POSTHOG_PROJECT_API_KEY, POSTHOG_HOST, TELEMETRY_ENABLED } from './config';
+import { PORT, SENTRY_DSN, NODE_ENV, MONGO_URL, SITE_URL } from './config';
 import { apiLimiter } from './helpers/rateLimiter';
+import { createTerminus } from '@godaddy/terminus';
 
 const app = express();
 
 Sentry.init({
-	dsn: SENTRY_DSN,
-	tracesSampleRate: 1.0,
-	debug: NODE_ENV === 'production' ? false : true,
-	environment: NODE_ENV
+  dsn: SENTRY_DSN,
+  tracesSampleRate: 1.0,
+  debug: NODE_ENV === 'production' ? false : true,
```

Comment:
Why use `===` here? I think the three equlas check if the operands are equal not the string themselves

Reply:
Oh, I did not change this line.

I think `===` checks for value and type equality.

Reply:
@reginaldbondoc is right here. We should be using `===` in most if not all cases.

Reply:
ack